cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -107,6 +107,12 @@ service SeismicAPI {
      */
     rpc SearchSeismicStores (SearchSeismicStoresRequest) returns (stream SeismicStore) {}
 
+    /**
+    * Inspect the seismicstore resulting from a possibly failed ingestion. Useful for 
+    * debugging failed ingestions.
+    */
+    rpc InspectIngestion (InspectIngestionRequest) returns (SeismicStore) {}
```

Comment:
Would it be worth returning a message that wraps the seismic store so that we can add any extra ingestion worker info if needed?

Reply:
Good idea! We could for example render the coordinate transformation into the message?

Reply:
Putting the coordinate transformation off until later if we need it.

Reply:
Putting the coordinate transformation off until later if we need it.